go.mod

```
@@ -3,7 +3,7 @@ module github.com/giantswarm/releases
 go 1.13
 
 require (
-	github.com/Masterminds/semver/v3 v3.1.0
+	github.com/blang/semver v3.5.0+incompatible
```

Can we use the latest? It is compatible with go modules.
```suggestion
	github.com/blang/semver/v4 v4.0.0
```

---

The use case is while crafting a new release while not having a final release of the provider operator yet.
For example, we are currently crafting release `14.0.0` for azure, that has a request for azure operator > 5.2.0.
We set a reference to the latest master in the release PR for now, and the version is 5.2.2-dev.
Does it make sense?

---

Ok that makes more sense. I was trying to understand why we'd ever request a dev version, now I get that it's the version which is being _compared to_ the request.

---

That comparison behavior is probably incorrect anyway though -- maybe you should make an issue with `masterminds` as well